Prog/SF_2010_14/Acs_sf_2010_14_dc.sas

```diff
@@ -34,11 +34,11 @@
   table_list = 
   	B00001 B00002 B01002 B01003 
 	B01001 B01001B B01001C B01001D B01001E B01001F B01001G B01001H B01001I
-	B02001 B03002 B05002 B06002 B06007 B06009 B07012 B09001
+	B02001 B03001 B03002 B05002 B05003I B06002 B06007 B06009 B07012 B09001
     B06004 B06004B B06004C B06004D B06004E B06004F B06004G B06004H B06004I
```

Comment:
I did--but for some reason they're syncing to an old branch that I thought I deleted and renamed to develop-#26. The original branch name is update_acs_table_list. Are you able to merge from that one?